packages/components/psammead-visually-hidden-text/src/index.jsx

```diff
@@ -1,6 +1,6 @@
-import styled from 'styled-components';
+import styled, { css } from 'styled-components';
 
-const VisuallyHiddenText = styled.span`
+export const VisuallyHiddenTextStyle = css`
```

Comment:
It might help to know that we don't need to use a `css` tagged template literal here, because we're dealing with a standard string.

https://styled-components.com/docs/api#css

This could be re-written as:

```js
export const visuallyHiddenTextStyle = `
  clip-path: inset(100%);
  clip: rect(1px, 1px, 1px, 1px);
  height: 1px;
  overflow: hidden;
  position: absolute;
  width: 1px;
  margin: 0;
`;
```

_Note: You may also notice this example uses camelCase, because we're not exporting a React component (typically written in TitleCase), but a string._